docs/user-guide/station-authentication-modes.md

```diff
@@ -0,0 +1,127 @@
+
+Supported values for 'CLIENT_CERTIFICATE_MODE' environment variables are:
+
+- '0' or 'NoCertificate' (client certificate is not required and will not be requested from clients)
+- '1' or 'AllowCertificate' (client certificate will be requested; however, authentication will not fail if a certificate is not provided by the client)
+- '2' or 'RequireCertificate' (client certificate will be requested, and the client must provide a valid certificate for authentication to succeed)
+
+### Importing 'tc.crt/tc.key/cups.crt/cups.key' in bundled 'LoRaBasicsStationModule'
+
+If you are making use of the bundled 'LoRaBasicsStationModule', it's possible to import a client certificate (.crt + .key files) in the module itself.
+
+The default path where the files will be searched is '/var/lorastarterkit/certs/'. As described in the previous section, it is possible to bind a folder on the host os to the one mentioned above.
+
+The default path of the tc.crt/tc.key/cups.crt/cups.key file can be overridden by using the related environment variable (i.e.: setting any of **'TC_CRT_PATH'**, **'TC_KEY_PATH'**, **'CUPS_CRT_PATH'**, **'CUPS_KEY_PATH'** to '/var/otherfolder/my.crt' will make the module copy the my.crt file to a *.crt in the LoRa Basics™ Station working directory).
+
+### Providing a list of allowed client thumbprints for connection
+
+Server side, the validation of the certificate is happening by comparing the thumbprint of the certificate provided for authentication with a list of allowed thumbprints to be stored in the Concentrator Twin (more information on 'clientThumbprint' property of Twin in [related ADR](https://azure.github.io/iotedge-lorawan-starterkit/dev/adr/006_cups/))
+
+When using the provided **Cli-LoRa-Device-Provisioning** tool to provision a concentrator device to IoT Hub, you can pass the **'--client-certificate-thumbprint'** option to specify the thumbprint of an allowed certificate.
+
+If you can't use the tool, when creating the Concentrator device in IoT Hub, all you need to do is to add a desired property named '**clientThumbprint**' (being an array of allowed certificate thumbprints) and specify the thumbprint in this array.
+
+## Self-signed certificates generator (for tests only)
+
+This starter kit is providing a [BasicStation Certificates Generation](https://github.com/Azure/iotedge-lorawan-starterkit/tree/dev/Tools/BasicStation-Certificates-Generation) tool for helping its users to generate LoRaWAN Network Server certificates and Basics Station certificates for **testing** secure communication between a Basics Station client and the CUPS/LNS Protocol Endpoint in Network Server.
+
+The starter kit and, therefore, this tool are expecting the same certificate sets for both CUPS and LNS endpoints.
+
+In a production environment, you should not use certificates provided by this tool, but generate and sign certificates with a trusted root authority.
```

Comment:
This makes it show up as a nice warning: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types
```suggestion
!!! danger "Only for testing"

    In a production environment, you should not use certificates provided by this tool, but generate and sign certificates with a trusted root authority.
```